src/Styleguide/Elements/__stories__/Typography.story.tsx

```diff
@@ -1,38 +1,35 @@
 import { storiesOf } from "@storybook/react"
 import React from "react"
-import { Theme } from "../../theme"
+import { Theme, Display, Sans, Serif } from "@artsy/palette"
+import { Col } from "../Grid"
 import { withInfo } from "@storybook/addon-info"
-import { Display, Sans, Serif } from "../Typography"
 
 const stories = storiesOf("Styleguide/Typography", module)
 
 stories
+  .addDecorator(storyFn => {
+    return <Theme>{storyFn()}</Theme>
+  })
```

Comment:
Could this go into our own storiesOf wrapper?

Reply:
Oh, yeah! `storiesOf("Styleguide")`? Would that apply to even the nested stories?

Reply:
I'll give it a try

Reply:
I mean something like:

```js
function storiesOf(desc, mod) {
  return require(“storybooks”).storiesOf(desc, mod).addDecorator(...)
}
```